proto/buf/alpha/registry/v1alpha1/user.proto

```diff
@@ -59,8 +59,7 @@ message OrganizationUser {
 // These preferences are stored in the database so the language and plugins
 // are pre-selected when the user navigates back to the Generated SDKs page from any device or browser.
 message UserPluginPreference {
-  // The id of the user.
-  string user_id = 1;
+  reserved 1;
```

Comment:
We should probably be consistent and reserve both field number and name. E.g.,

https://github.com/bufbuild/buf/blob/0b24de2535e571ea33a0dbb280b1407c0fbb764e/proto/buf/alpha/registry/v1alpha1/organization.proto#L234-L235